modules/strelka/1.0/strelka.smk

```diff
@@ -0,0 +1,213 @@
+        #snp = CFG["dirs"]["strelka"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/results/variants/somatic.snvs.vcf.gz",
+        #indels = CFG["dirs"]["strelka"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/results/variants/somatic.indels.vcf.gz"
+    log:
+        stdout = CFG["logs"]["strelka"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/strelka_run.stdout.log",
+        stderr = CFG["logs"]["strelka"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/strelka_run.stderr.log"
+    params:
+        opts = CFG["options"]["strelka"]
+    conda:
+        CFG["conda_envs"]["strelka"]
+    threads:
+        CFG["threads"]["strelka"]
+    resources: 
+        mem_mb = CFG["mem_mb"]["strelka"]
+    shell:
+        op.as_one_line("""
+        {input.runwf} -j {threads} 
+        {params.opts} 
+        >{log.stdout} 2> {log.stderr}
+        """)
+
+
+rule _strelka_unzip:
+    input:
+        vcf_dir = rules._strelka_run.output.vcf_dir
+        #vcf = CFG["dirs"]["strelka"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/results/variants/{var_type}.vcf.gz"
+    output:
+        vcf = CFG["dirs"]["filtered"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/{var_type}.vcf"
+    shell:
+        "zcat {input.vcf_dir}/{wildcards.var_type}.vcf.gz > {output.vcf}"
+
+
+rule _strelka_filter:
+    input:
+        vcf = rules._strelka_unzip.output.vcf
+    output:
+        vcf = CFG["dirs"]["filtered"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/{var_type}.filt.vcf"
+    shell:
+        op.as_one_line("""
+        grep '^#' {input.vcf} > {output.vcf} && 
+        awk '{{ if ($7 == \"PASS\") {{ print $0 }} }}' {input.vcf} >> {output.vcf}
+        """)
+
+
+# Symlinks the final output files into the module results directory (under '99-outputs/')
+rule _strelka_output_vcf:
+    input:
+        vcf = rules._strelka_filter.output.vcf
+    output:
+        vcf = CFG["dirs"]["outputs"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/{var_type}.vcf"
+    run:
+        op.relative_symlink(input.vcf, output.vcf)
+
+
+def _get_strelka_output(wildcards):
+    CFG = config["lcr-modules"]["strelka"]
+
+    if wildcards.pair_status == "no_normal":
+        vcf_files = "variants"
+    else:
+        vcf_files = ["somatic.snvs", "somatic.indels"]
+    vcf = expand(rules._strelka_output_vcf.output.vcf, var_type = vcf_files, **wildcards)
+    return vcf
+
+
+rule _strelka_dispatch:
+    input: 
+        vcf = _get_strelka_output
+    output:
+        dispatched = touch(CFG["dirs"]["outputs"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.dispatched")
+
+
+# Generates the target sentinels for each run, which generate the symlinks
+rule _strelka_all:
```

Comment:
You might want to expose both the filtered and unfiltered VCF files in the `99-outputs` directory. But when we implement variant annotation, we can just annotate the filtered VCF because it would take too much time otherwise.